src/lib/formValidator.ts

```diff
@@ -0,0 +1,66 @@
+import {
+  IFieldValidator,
+  IFieldValidationResponse,
+  fieldValidatorLookup,
+} from "./fieldValidators";
+
+export interface IFormData {
+  [key: string]: string;
+}
+
+export interface IFieldValidatorLookup {
+  [key: string]: IFieldValidator;
+}
+
+export interface IFormValidationResponse {
```

Comment:
Again does this need to be an interface. It could be Record<string, IFieldValidationResponse > or if you wanted the type I would be tempted to define it as:

```typescript
export type IFormValidationResponse = Record<string, IFieldValidationResponse>;
```